client/influxdb.go

```diff
@@ -145,6 +145,27 @@ func (c *Client) Ping() (time.Duration, string, error) {
 	return time.Since(now), version, nil
 }
 
+func (c *Client) Dump(db string) (*http.Response, error) {
```

Comment:
I think we should return a io.ReadCloser so that anyone else consuming our library wouldn't have to add the `http` package if they wanted to declare a variable to receive the response.